docs/src/pages/components/masonry/masonry.md

```diff
@@ -0,0 +1,69 @@
+---
+title: React Masonry component
+components: Masonry, MasonryItem
+githubLabel: 'component: Masonry'
+---
+
+# Masonry
+
+<p class="description">Masonry lays out contents of different sizes as blocks of the same width and variable height with configurable gaps.</p>
+
+Masonry maintains a list of content blocks with a consistent width but variable height.
+The contents are ordered by row.
+If a row is already filled with the specified number of columns, the next item starts another row, and it is added to the shortest column.
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": true}}
```

Comment:
**Off-topic** (not related to this PR directly)

I wonder if we should render the design kit links here. Same question for the Stack component.

Reply:
I think it makes sense to have both components in the design kits - detachable components from where designers add their content. This way, the layer name "Stack" or "Masonry" respectively could be easily picked up by developers to know which API to use. Also, starting-point examples could be nice to have to understand how it works.

What do you think about this idea @danilo-leal

Reply:
It's not clear to me how would I use that on the design tool. Figma would probably be better because of auto-layout but what about the other ones?! Is it a possibility to make a quick prototype for us to get a sense of it? Asking because I don't usually make into a component layout components such as these (stack, masonry, grid, etc).

Reply:
Ok, I'm removing these links, for now